R/data-objects.R

```diff
@@ -19,28 +19,44 @@
 #'  exists. Defaults to `FALSE`.
 #'
 #' @return (Invisibly) the HTTP response.
+#' @seealso
+#'  [iget()] for obtaining files,
+#'  [ireadRDS] for obtaining R objects from iRODS,
```

Comment:
Add parentheses: `[ireadRDS()]` and `[saveRDS()]`

Reply:
done